addons/gdUnit4/src/core/parse/GdScriptParser.gd

```diff
@@ -385,8 +389,21 @@ func parse_arguments(input: String) -> Array[GdFunctionArgument]:
 	var in_function := false
 	while current_index < len(input):
 		token = next_token(input, current_index)
+		# fallback to not end in a endless loop
+		if token == TOKEN_NOT_MATCH:
+			var error : = """
+				Parsing Error: Invalid token at pos %d found.
+				Please report this error!
+				source_code:
+				--------------------------------------------------------------
+				%s
+				--------------------------------------------------------------
+			""".dedent() % [current_index, input]
+			push_error(error)
+			current_index += 1
+			continue
 		current_index += token._consumed
-		if token == TOKEN_SPACE:
+		if token == TOKEN_SPACE or token == TOKEN_TABULATOR or token == TOKEN_NEW_LINE:
```

Comment:
Adding a convenience function `ist_skippable()` to simplify this code to
`if token.is_skippable():`